executor/src/witgen/symbolic_evaluator.rs

```diff
@@ -101,11 +107,13 @@ impl<'a> SymbolicVariables for SymbolicEvaluator<'a> {
 
 impl<'a> WitnessColumnNamer for SymbolicEvaluator<'a> {
```

Comment:
We need to get rid of this whole trait by just changing the key id of AffineExpression to &PolynomialReference.

Reply:
This is in a different PR?